src/schema/posts.ts

```diff
@@ -2215,6 +2221,35 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       }
       return { _: true };
     },
+    clickbaitPost: async (_, { id }: { id: string }, ctx: AuthContext) => {
+      const { contentQuality }: Pick<Post, 'contentQuality'> = await ctx.con
+        .getRepository(Post)
+        .findOneOrFail({
+          where: { id },
+          select: ['contentQuality'],
+        });
+
+      const is_clickbait_probability =
+        contentQuality.is_clickbait_probability || 0.0;
+      const clickbaitTitleProbabilityThreshold =
+        remoteConfig.vars.clickbaitTitleProbabilityThreshold || 1.0;
+
+      // We +/- 10 as it is a number that is above the natural range from Yggdrasil,
+      // so it will be easy to identify if the value was changed by a user.
+      const newProbability =
+        is_clickbait_probability > clickbaitTitleProbabilityThreshold
+          ? is_clickbait_probability - 10.0
+          : is_clickbait_probability + 10.0;
+
+      await ctx.con.getRepository(Post).update(
+        { id },
+        {
+          contentQuality: {
+            is_clickbait_probability: newProbability,
+          },
```

Comment:
Check with Denis if Yggdrasil can send this again with post update which will overwrite the set value, we might need to update `postUpdated` to ignore yggdrasil value in there?

Reply:
cc @denisb0

Reply:
Yes, any values sent with published content, can be sent with update. From Yggdrasil side there is only publish action, no difference between first publish and subsequent update.
It very rarely happens in the course of normal operation, but backfilling the data may be the issues.

Reply:
I added a flag `manual_clickbait_probability` that worker checks for and if exists, keeps value that is set in database.